src/renderer/views/Search/Search.js

```diff
@@ -98,23 +98,24 @@ export default defineComponent({
         return search.query === payload.query && searchFiltersMatch(payload.searchSettings, search.searchSettings)
       })
 
-      this.shownResults = []
-      this.isLoading = true
-
       if (sameSearch.length > 0) {
+        // No loading effect needed here, only rendered result update
+
         // Replacing the data right away causes a strange error where the data
         // Shown is mixed from 2 different search results.  So we'll wait a moment
         // Before showing the results.
         setTimeout(this.replaceShownResults, 100, sameSearch[0])
```

Comment:
We don't need this timeout anymore, now that it updates the list correctly.
```suggestion
        this.replaceShownResults(sameSearch[0])
```